subworkflows/annotate.nf

```diff
@@ -85,7 +92,7 @@ workflow ANNOTATE {
             eggnog_data_dir
         )
 
-        interproscan_annotations = IPS.out.ips_annontations.collectFile(
+        interproscan_annotations = IPS.out.ips_annotations.collectFile(
```

Comment:
![image](https://github.com/user-attachments/assets/ae1728cf-050b-4709-b4cc-2c85d381090f)

thanks for fixing that